python-main.js

```diff
@@ -548,6 +560,7 @@ function web_editor(config) {
         $('#editor').on('drop', doDrop);
         // Focus on the element with TAB-STATE=1
         $("#command-download").focus();
+        
```

Comment:
Was this blank line introduced by accident? If so, remove it so we have less differences from the upstream repository